tests/migration_test.go

```diff
@@ -2573,6 +2573,96 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				Entry("[sig-compute][test_id:3241]cancel a migration right after posting it", false),
 				Entry("[sig-compute][test_id:3246]cancel a migration with virtctl", true),
 			)
+
+			Context("[Serial]when target pod cannot be scheduled and is suck in Pending phase", func() {
+
+				var nodesSetUnschedulable []string
+
+				BeforeEach(func() {
+					By("Keeping only one schedulable node")
+					schedulableNodes := libnode.GetAllSchedulableNodes(virtClient).Items
+					Expect(len(schedulableNodes)).To(BeNumerically(">=", 1))
```

Comment:
`Expect(schedulableNodes).NotTo(And(HaveLen(0),HaveLen(1)))` will give you better message.

@EdDev I guess this is why you proposed the linter.

Reply:
Done, thanks